src/content_mapblock.cpp

```diff
@@ -18,6 +18,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 */
 
 #include "content_mapblock.h"
+#include "content_mapblock_private.h"
```

Comment:
Not sure should it be private. Maybe it’s better to make the class public.
Anyway, for me it’s handier to have that in a separate file, at least for the cleanup stage. After that I may merge it into either `.cpp` or `.h`, if you consider that necessary.